src/shared/parseReleaseNotes.ts

```diff
@@ -6,37 +6,56 @@
  */
 
 import { marked } from 'marked';
+import * as semver from 'semver';
 
 const parseReleaseNotes = (notes: string, version: string, baseUrl: string): marked.Token[] => {
   let found = false;
+  let findClosest = false;
+  let closestVersion: string;
+  const versions = [];
 
   const parsed = marked.lexer(notes);
 
-  // https://stackoverflow.com/a/6969486
-  const escapeRegExp = (string: string): string => {
-    return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
-  };
+  let tokens: marked.Token[];
+
+  const findVersion = (desiredVersion: string): void => {
+    tokens = parsed.filter((token) => {
+      // TODO: Could make header depth (2) a setting in oclif.info.releasenotes
+      if (token.type === 'heading' && token.depth === 2) {
+        const coercedVerion = semver.coerce(token.text).version;
+
+        // First time through, build an array of versions
+        // We will use this to find the closest patch if passed version is not found
+        if (!findClosest) versions.push(coercedVerion);
 
-  const regexp = new RegExp(`\\b${escapeRegExp(version)}\\b`);
+        if (coercedVerion === desiredVersion) {
+          found = true;
 
-  const tokens = parsed.filter((token) => {
-    // TODO: Could make header depth (2) a setting in oclif.info.releasenotes
-    if (token.type === 'heading' && token.depth === 2) {
-      if (regexp.exec(token.text)) {
-        found = true;
+          return token;
+        }
 
+        found = false;
+      } else if (found === true) {
         return token;
       }
+    });
 
-      found = false;
-    } else if (found === true) {
-      return token;
+    if (!tokens.length) {
+      if (findClosest === false) {
+        findClosest = true;
+
+        const semverRange = `${semver.major(version)}.${semver.minor(version)}.x`;
+        // Version was not found, try again with the closest patch version
+        closestVersion = semver.maxSatisfying(versions, semverRange) as string;
```

Comment:
```suggestion
        closestVersion = semver.maxSatisfying<string>(versions, semverRange);
```

This lets you pass in the output type rather than asserting it.  It's up to the library to constrain what types it accepts.